tss-esapi/tests/integration_tests/error_tests/return_code_tests/base_tests.rs

```diff
@@ -168,70 +168,108 @@ fn test_invalid_conversions() {
     assert_eq!(
         BaseReturnCode::try_from(tss_invalid_base_rc as u16),
         Err(Error::WrapperError(WrapperErrorKind::InvalidParam)),
-        "Converting invalid base resposne code did not produce the expected error",
+        "Converting invalid base response code did not produce the expected error",
```

Comment:
:thinking: weird that codespell did not pick this up... I need to check this out locally...

Reply:
Yeah, locally it correctly identified this issue. I need to check it out why GitHub Action did not flag it (and others) earlier :/